src/Tizen.NUI/src/public/Theme/Theme.cs

```diff
@@ -224,21 +228,22 @@ internal ResourceDictionary Resources
         /// Gets a style of given style name.
         /// </summary>
         /// <param name="styleName">The string key to find a ViewStyle.</param>
-        /// <returns>Founded style instance.</returns>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <returns>Founded style instance if the style name is found, otherwise null.</returns>
```

Comment:
Founded -> Found,   name is found -> name has been found

Please replace other ocurrences of "founded" as well

Reply:
Done.